docs/guides/conda.rst

```diff
@@ -8,16 +8,171 @@ and need them installed when building their documentation.
+   If your project needs it, request that we enable the ``CONDA_APPEND_CORE_REQUIREMENTS``
+   :ref:`feature flag <guides/feature-flags:Feature Flags>`.
+
+.. _this GitHub issue for discussion: https://github.com/readthedocs/readthedocs.org/issues/3829
+.. _exporting a conda environment: https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#sharing-an-environment
+
+Effective use of channels
+-------------------------
+
+Conda packages are usually hosted on https://anaconda.org/, a registration-free artifact archive
+maintained by Anaconda Inc. Contrary to what happens with the Python Package Index,
+different users can potentially host the same package in the same repository,
+each of them using their own **channel**. Therefore, when installing a conda package,
+conda also needs to know which channels should use, and which ones take precedence.
+
+If not specified, conda will use ``defaults``, the channel maintained by Anaconda Inc.
+and subject to `Anaconda Terms of Service`_. It contains well-tested versions of the most widely used
+packages. However, some packages are not available on the ``defaults`` channel,
+and even if they are, they might not be on their lastest versions.
+
+As an alternative, there are channels maintained by the community that have a broader selection
+of packages and more up-to-date versions of them, the most popular one being ``conda-forge``.
+
+To use the ``conda-forge`` channel when specifying your project dependencies, include it in the list
+of ``channels`` in ``environment.yml``, and conda will rank them in order of appearance.
+To maximize compatibility, we recommend putting ``conda-forge`` above ``defaults``.
+
+.. tip:: If you want to opt out the ``defaults`` channel completely, replace it by ``nodefaults``
+   in the list of channels. See `the relevant conda docs`_ for more information.
+
+.. _Anaconda Terms of Service: https://www.anaconda.com/terms-of-service
+.. _the relevant conda docs: https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html?highlight=nodefaults#creating-an-environment-file-manually
 
-* Environment Creation (``conda env create``)
-* Dependency Installation (Sphinx)
+Mixing conda and pip packages
+-----------------------------
+
+There are valid reasons to use pip inside a conda environment: some dependency
+might not be avaliable yet as a conda package in any channel,
+or you might want to avoid precompiled binaries entirely.
+In either case, it is possible to specify the subset of packages
+that will be installed with pip in the `environment.yml` file. For example:
+
+.. code-block:: yaml
+
+    name: rtd38
+    channels:
+      - defaults
+      - conda-forge
+    dependencies:
+      - rasterio==1.2
+      - python=3.8
+      - pytorch-cpu=1.7
+      - pip>=20.1  # pip is needed as dependency
+      - pip:
+        - black==20.8b1
+    prefix: /home/juanlu/.mambaforge/envs/rtd38
```

Comment:
think we could remove the prefix key from the examples

Reply:
Good idea, done!